python/dgl/nn/pytorch/gt/spatial_encoder.py

```diff
@@ -140,107 +142,63 @@ def __init__(self, num_kernels, num_heads=1, max_node_type=1):
         self.gaussian_stds = nn.Embedding(1, num_kernels)
         self.linear_layer_1 = nn.Linear(num_kernels, num_kernels)
         self.linear_layer_2 = nn.Linear(num_kernels, num_heads)
-        if max_node_type == 1:
-            self.mul = nn.Embedding(1, 1)
-            self.bias = nn.Embedding(1, 1)
-        else:
-            self.mul = nn.Embedding(max_node_type + 1, 2)
-            self.bias = nn.Embedding(max_node_type + 1, 2)
+        # default mul/bias at position 0 (no node type given)
+        # src mul/bias at position [1 (pad) ~ max_node_type+1]
+        # tgt mul/bias at position [max_node_type+2 (pad) ~ 2*max_node_type+2]
```

Comment:
I don't understand the comments here.

Reply:
updated the comments

Reply:
Thanks for the clarification. Will users ever need to have a module instantiation that can both handle typed and untyped nodes? When type information is not available, isn't this simply `max_node_type` = 1?

Reply:
Yes, e.g., the model is pre-trained on dataset where nodes have types but fine-tuned on dataset where node types are not available. Although in this scenario the model uses different beta and gamma during pre-training and fine-tuning (only default beta and gamma used) phases, a single model handle both types of data can avoid extra burden on resetting specific modules/parameters when loading pre-trained models.